examples/rust/file_transfer/tests/tiny_test_file.bin

```diff
@@ -0,0 +1,2 @@
+E)�E�u�W��q�w�
```

Comment:
it would be better to not have this committed to the repo - various security linters complain about file formats they don't recognize. maybe the test can generate a tempfile before trying a transfer.

Reply:
Okay, sure. I'll remove the need for the static binary files. That's good to know about the linters.

Reply:
I've updated the PR. Thanks!